pylibui/libui/separator.py

```diff
@@ -0,0 +1,46 @@
+"""
+ Python wrapper for libui.
+
+"""
+
+import ctypes
+from . import clibui
+
+
+class uiSeparator(ctypes.Structure):
+    """Wrapper for the uiSeparator C struct."""
+
+    pass
+
+
+def uiSeparatorPointer(obj):
+    """
+    Casts an object to uiSeparator pointer type.
+
+    :param obj: a generic object
+    :return: uiSeparator
+    """
+
+    return ctypes.cast(obj, ctypes.POINTER(uiSeparator))
+
+
+# - uiSeparator *uiNewHorizontalSeparator(void);
+def uiNewHorizontalSeparator():
+    """
+    Return a horizonal separator.
+
+    :return: HorizontalSeparator
```

Comment:
Also this line is not correct. Please compare with the other files in this directory. Use the same pattern.